powerauth-client-model/src/main/java/com/wultra/security/powerauth/client/model/request/CreateCallbackUrlRequest.java

```diff
@@ -43,8 +45,8 @@ public class CreateCallbackUrlRequest {
     @NotBlank
     private String name;
 
-    @NotBlank
-    private String type;
+    @NotNull
+    private CallbackUrlType type;
```

Comment:
🤔 Shouldn't it be reflected in `WebServices-Methods.md`?

Reply:
sure, added now